internal/runtimehandlerhooks/high_performance_hooks_test.go

```diff
@@ -766,6 +767,81 @@ var _ = Describe("high_performance_hooks", func() {
 			env := g.Config.Process.Env
 			Expect(env).To(ContainElements("OPENSHIFT_ISOLATED_CPUS=1-2", "OPENSHIFT_SHARED_CPUS=3-4"))
 		})
+
+		Describe("Exec CPU Affinity", func() {
+			var sbNoShared *sandbox.Sandbox
+			BeforeEach(func() {
+				sbox := sandbox.NewBuilder()
+				createdAt := time.Now()
+				sbox.SetCreatedAt(createdAt)
+				sbox.SetID("sandboxID")
+				sbox.SetName("sandboxName")
+				sbox.SetLogDir("test")
+				sbox.SetShmPath("test")
+				sbox.SetNamespace("")
+				sbox.SetKubeName("")
+				sbox.SetMountLabel("test")
+				sbox.SetProcessLabel("test")
+				sbox.SetCgroupParent("")
+				sbox.SetRuntimeHandler("")
+				sbox.SetResolvPath("")
+				sbox.SetHostname("")
+				sbox.SetPortMappings([]*hostport.PortMapping{})
+				sbox.SetHostNetwork(false)
+				sbox.SetUsernsMode("")
+				sbox.SetPodLinuxOverhead(nil)
+				sbox.SetPodLinuxResources(nil)
+				err = sbox.SetCRISandbox(sbox.ID(), make(map[string]string), map[string]string{}, &types.PodSandboxMetadata{})
+				Expect(err).ToNot(HaveOccurred())
+				sbox.SetPrivileged(false)
+				sbox.SetHostNetwork(false)
+				sbox.SetCreatedAt(createdAt)
+				sbNoShared, err = sbox.GetSandbox()
+				Expect(err).ToNot(HaveOccurred())
+			})
```

Comment:
Thanks for adding the unit tests. Instead of nesting the Describe and duplicating the BeforeEach, it would be cleaner to restructure the original "Precreate hook" test to use the "Context"/"BeforeEach" pattern (see other tests in this file) and pass in the crioannotations for each test, which seems to be the only thing that differs here.